### ISSUE TITLE
Edit Receipt Layout

### ISSUE THREAD

**Issue by willymarlian** [hiramvillarreal/iotpos#6]
I understand the script print-cups.cpp and can edit it, please tell me the command to compile this script to only modify the print receipt section. 

I think it will be better if there is a GUI tool to update the print layout.

Thanks.

**Comment by hiramvillarreal** [comment]
Did you already try to modify the file and ther recompile and reinstalll the entire program and check the results?
Hugs.

**Comment by willymarlian** [comment]
Yes. I did. I did make and make install, everything works as expected but it takes a while because it was compiled all scripts. I was thinking to compile only the file printer-cups so it will speed ip the process. Can you help me with that?

**Comment by willymarlian** [comment]
I have another problem, when I printed receipt for 30 items it was ok, but if the items more than 30, it will be cropped. Do you know how to solve this problem? Does this problem has something to do with paper length or buffer printer limited?

**Comment by hiramvillarreal** [comment]
Hi, I never notice letme check, are you using a serial printer via gpio? or a CUPS printer?

**Comment by willymarlian** [comment]
I'm using CUPS printer because I cannot setup serial printer with my Thermal Printer, maybe the thermal printer it is not compatible. Thanks for checking.

**Comment by Shaggs** [comment]
Looking to close old issues has this been resolved?

**Comment by hiramvillarreal** [comment]
Not resolved get working on...

**Comment by Shaggs** [comment]
ill have a look at this once i have sorted out the Credit print issue. Sounds like it could be a buffer issue or paper size issue.
